File: client_lourd/src/app/classes/commands/data/draw-rectangle-data.ts
import { Vec2 } from '@app/classes/vec2';
export class DrawRectangleData {
    strokeColor: string;
    fillColor: string;
    size: number;
    withStroke: boolean;
    withFill: boolean;
    isEven: boolean;
    beginning: Vec2;
    end: Vec2;
    text: string;
    textColor: string;
}

File: client_leger/app/src/main/java/com/client_leger/colorimage/drawing/data/DrawRectangleData.kt
package com.client_leger.colorimage.drawing.data

data class DrawRectangleData(
    val strokeColor : String,
    val fillColor: String,
    val size : Int,
    val withStroke: Boolean,
    val withFill : Boolean,
    val beginning: Vec2,
    val end: Vec2,
    val isEven: Boolean,
    val text: String?,
    val textColor: String?
) : CommandData()
